frontend/src/components/apps/function-detail.tsx

```
@@ -13,20 +14,170 @@ import {
+
+  if (!isParameterSchema(parametersSchema)) {
+    return null;
+  }
+
+  if (parametersSchema.type !== "object") {
+    return parametersSchema;
+  }
+
+  const result = { ...parametersSchema };
+  const visible = result.visible || [];
+  delete result.visible;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)